composer.json

```diff
@@ -11,7 +11,9 @@
         }
     ],
     "require-dev": {
-        "bcgov/wordpress-scripts": "2.1.2"
+        "bcgov/wordpress-scripts": "2.1.2",
```

Comment:
i'm pretty sure this is the new file for wordpress-scripts https://github.com/bcgov/wordpress-utils, however i see we haven't actually updated the composer.json to reflect this.